modules/vaos/spec/routing/vaos_routing_spec.rb

```diff
@@ -2,83 +2,196 @@
 
 require 'rails_helper'
 
-RSpec.describe 'routes for Session', type: :routing do
-  it 'routes to the appointment requests index' do
-    expect(get('/v0/vaos/appointment_requests')).to route_to(
-      format: :json,
-      controller: 'vaos/v0/appointment_requests',
-      action: 'index'
-    )
-  end
+RSpec.describe 'VAOS routing configuration', type: :routing do
+  # Temporarily supporting '/v0/vaos' but also '/vaos/v0'
+  ['/v0/vaos', '/vaos/v0'].each do |partial_route|
```

Comment:
I think it might be less complex and easier to pull out later if you just copied this file and renamed it something like `v0_vaos_routing_spec.rb`

Reply:
I guess it could make maintenance a pain, how long will these temp routes be around?

Reply:
not long, no more than 1-2 weeks

Reply:
I do want to stress a little bit of urgency on this one which is making me nervous -- we have to get this in before the production deploy -- that or revert the last two PRs that have since been merged. Sorry for not testing these properly.